subsys/bluetooth/mesh/Kconfig

```diff
@@ -9,6 +9,7 @@ menuconfig BT_MESH
 	select TINYCRYPT_AES
 	select TINYCRYPT_AES_CMAC
 	select BT_HOST_CCM
+	select BT_EXT_ADV
```

Comment:
Where do you plan to put the updated defaults for "BT_MESH_RELAY_ADV_SETS, BT_MESH_ADV_EXT_GATT_SEPARATE", downstream?